portal-impl/src/com/liferay/portlet/PortalPreferencesImpl.java

```diff
@@ -310,7 +349,12 @@ protected boolean isCausedByStaleObjectException(Throwable t) {
 		return false;
 	}
 
-	protected void retryableStore(Callable<?> callable) throws Throwable {
+	protected void retryableStore(
+			Callable<?> callable, String namespace, String key)
```

Comment:
Don't do this namespace handling, just call super.getValues() which does not have namespace.